x/group/client/testutil/tx.go

```diff
@@ -2347,3 +2432,129 @@ func (s *IntegrationTestSuite) createCLIProposal(groupPolicyAddress, proposer, s
 
 	return testutil.WriteToNewTempFile(s.T(), string(bz)).Name()
 }
+
+func (s *IntegrationTestSuite) createAccounts(quantity int) []string {
+	val := s.network.Validators[0]
+	clientCtx := val.ClientCtx
+	accounts := make([]string, quantity)
+
+	for i := 1; i <= quantity; i++ {
+		memberNumber := uuid.New().String()
+
+		info, _, err := clientCtx.Keyring.NewMnemonic(fmt.Sprintf("member%s", memberNumber), keyring.English, sdk.FullFundraiserPath,
+			keyring.DefaultBIP39Passphrase, hd.Secp256k1)
+		s.Require().NoError(err)
+
+		pk, err := info.GetPubKey()
+		s.Require().NoError(err)
+
+		account := sdk.AccAddress(pk.Address())
+		accounts[i-1] = account.String()
+
+		_, err = banktestutil.MsgSendExec(clientCtx, val.Address, account,
+			sdk.NewCoins(sdk.NewCoin(s.cfg.BondDenom, sdk.NewInt(1000))),
+			s.commonFlags...,
+		)
+		s.Require().NoError(err)
+	}
+	return accounts
+}
+
+func (s *IntegrationTestSuite) createGroupWithMembers(membersWeight, membersAddress []string) string {
+	s.Require().Equal(len(membersWeight), len(membersAddress))
+	val := s.network.Validators[0]
+	clientCtx := val.ClientCtx
+
+	membersValid := s.newValidMembers(membersWeight, membersAddress)
+	membersValidByte, err := membersValid.Marshal()
+	s.Require().NoError(err)
+
+	validMemberString := string(membersValidByte)
+	s.Require().NoError(err)
+
+	validMembersFile := testutil.WriteToNewTempFile(s.T(), validMemberString)
+	out, err := cli.ExecTestCLICmd(clientCtx, client.MsgCreateGroupCmd(),
+		append(
+			[]string{
+				val.Address.String(),
+				validMetadata,
+				validMembersFile.Name(),
+			},
+			s.commonFlags...,
+		),
+	)
+	s.Require().NoError(err, out.String())
+	var txResp sdk.TxResponse
+	s.Require().NoError(val.ClientCtx.Codec.UnmarshalJSON(out.Bytes(), &txResp), out.String())
+	return s.getGroupIdFromTxResponse(txResp)
+}
+
+func (s *IntegrationTestSuite) createGroupThresholdPolicyWithTokens(groupID string, threshold int, tokens int64) string {
```

Comment:
what about renaming this to `createGroupThresholdPolicyWithCoins` or `createGroupThresholdPolicyWithBalance`? this seems a bit closer to the vocabulary used throughout the sdk

Reply:
sure!!